roles/setup_pgd/tasks/pg_hba_config.yml

```diff
@@ -57,6 +56,52 @@
   run_once: true
   no_log: "{{ disable_logging }}"
 
+- name: Prepare hba non ssl list for user - pg_owner
+  ansible.builtin.set_fact:
+    pg_allow_ip_addresses: >-
+      {{ pg_allow_ip_addresses | default([]) + [
+        {
+          "contype": "host",
+          "users": pg_owner,
+          "source": node.private_ip + "/32",
+          "databases": pgd_cluster_database
+          },
+        {
+          "contype": "host",
+          "users": pg_owner,
+          "source": node.private_ip + "/32",
+          "databases": pg_database
+          }
+        ] }}
+  when: not pg_ssl
+  loop: "{{ pgd_cluster_nodes }}"
+  loop_control:
+    loop_var: node
+  run_once: true
+  no_log: "{{ disable_logging }}"
+
+- name: Prepare hba ssl list for user - pg_owner
+  ansible.builtin.set_fact:
+    pg_allow_ip_addresses: >-
+      {{ pg_allow_ip_addresses | default([]) + [
+        {
+          "users": pg_owner,
+          "source": node.private_ip + "/32",
+          "databases": pgd_cluster_database
+          },
+          {
+          "users": pg_owner,
+          "source": node.private_ip + "/32",
+          "databases": pg_database
+          }
+        ] }}
+  when: pg_ssl
+  loop: "{{ pgd_cluster_nodes }}"
+  loop_control:
+    loop_var: node
+  run_once: true
+  no_log: "{{ disable_logging }}"
+
```

Comment:
IIRC, we verified that this is not needed for configuration. Any reason for this to be here